contracts/registry/ReverseRegistrar.sol

```diff
@@ -49,14 +48,19 @@ contract ReverseRegistrar is Ownable, Controllable {
         _;
     }
 
+    function setDefaultResolver(address resolver) public override onlyOwner {
+        defaultResolver = NameResolver(resolver);
+    }
+
     /**
      * @dev Transfers ownership of the reverse ENS record associated with the
      *      calling account.
      * @param owner The address to set as the owner of the reverse record in ENS.
      * @return The ENS node hash of the reverse record.
      */
-    function claim(address owner) public returns (bytes32) {
-        return _claimWithResolver(msg.sender, owner, address(0x0));
+    function claim(address owner) public override returns (bytes32) {
+        require(address(defaultResolver) != address(0));
```

Comment:
We can remove this check, and just not allow it to be set to 0 in the first place.

Reply:
Do you mean not allow it to be set to 0 in the UI or you mean further down inside `claimForAddr`?